Packs/AgariPhishingDefense/Integrations/AgariPhishingDefense/AgariPhishingDefense.yml

```diff
@@ -0,0 +1,456 @@
+        YYYY-MM-ddTHH:mm:ss
+        N days
+        N hours
+        Example: 2020-05-01
+        2020-05-01T00:00:00
+        2 days
+        5 hours.
+      isArray: false
+      name: start_date
+      required: false
+      secret: false
+    - default: false
+      description: "The latest date time (UTC) a search should target (ISO 8601 format).\n\
+        Formats accepted: YYYY-MM-dd\nYYYY-MM-ddTHH:mm:ss\nN days\nN hours\nExample:\
+        \ 2020-05-01 \n2020-05-01T00:00:00\n2 days\n5 hours."
+      isArray: false
+      name: end_date
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '1'
+      description: To page through a collection of policy events.
+      isArray: false
+      name: page_id
+      required: false
+      secret: false
+    - default: false
+      description: 'A comma-delimited string that specifies the field ordering to
+        be applied to the response. Example: created_at DESC, id ASC.'
+      isArray: true
+      name: sort
+      required: false
+      secret: false
+    - default: false
+      description: A comma-delimited list of optional fields to add to the default
+        payload. Additional fields would add data in the entry context.
+      isArray: true
+      name: add_fields
+      required: false
+      secret: false
+    - default: false
+      description: A comma-delimited list of fields to remove from the default payload.
+        Limited fields would return limited data in entry context.
+      isArray: true
+      name: rem_fields
+      required: false
+      secret: false
+    - default: false
+      description: A comma-delimited list of fields to include in the payload. Limited
+        fields would return limited data in entry context.
+      isArray: true
+      name: fields
+      required: false
+      secret: false
+    - default: false
+      description: Search filters that can be applied to the response.
+      isArray: false
+      name: filter
+      required: false
+      secret: false
+    - default: false
+      description: Exclude policy types such as 'MessageAlert' or 'SystemAlert'.
+      isArray: false
+      name: exclude_alert_types
+      required: false
+      secret: false
+    - default: false
+      description: Find by policy name.
+      isArray: false
+      name: policy_name
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'Filter by policy action: ''deliver'', ''mark-spam'', ''move'',
+        ''inbox'', ''delete'' and ''none''.'
+      isArray: false
+      name: policy_action
+      predefined:
+      - deliver
+      - mark-spam
+      - move
+      - inbox
+      - delete
+      - none
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieves a list of policy events.
+    execution: false
+    name: apd-list-policy-events
+    outputs:
+    - contextPath: AgariPhishingDefense.Alert.alert_definition_name
```

Comment:
I suggest changing context paths per the Demisto standard.
For example, `alert_definition_name` to `AlertDefnitionName`.
Feel free to look at [this resource](https://xsoar.pan.dev/docs/integrations/context-and-outputs) for more data.

Reply:
@guykeller 
I think only dot and spaces are not supported for contextData if we get in API Response, it is fine to keep the _ (underscore).
We already confirmed the same.

cc: @anara123